src/query/plan/operator.cpp

```diff
@@ -1168,14 +1173,18 @@ auto ExpandFromVertex(const VertexAccessor &vertex, EdgeAtom::Direction directio
       memory);
 
   if (direction != EdgeAtom::Direction::OUT) {
-    auto edges = UnwrapEdgesResult(vertex.InEdges(view, edge_types)).edges;
+    auto edges_result = UnwrapEdgesResult(vertex.InEdges(view, edge_types));
+    context->number_of_hops += edges_result.expanded_count;
+    auto edges = std::move(edges_result.edges);
```

Comment:
Redundant line, you can just `std::move(edges_result.edges)` two lines further, there are more cases like this in the current version of the code.